internal/driver/flags.go

```diff
@@ -44,9 +45,33 @@ func (*GoFlags) String(o, d, c string) *string {
 	return flag.String(o, d, c)
 }
 
+type stringList []*string
+
+func (sl *stringList) Set(v string) error {
+	*sl = append(*sl, &v)
+	return nil
+}
+
+func (sl *stringList) Get() interface{} {
+	return sl
+}
+
+func (sl *stringList) String() string {
```

Comment:
For my understanding, where (other than tests) is this method used?

Reply:
This function is required to implement the flag.Value interface.  Link: https://golang.org/pkg/flag/#Value.